docs/advanced/tls_hardware_acceleration.md

```diff
@@ -59,14 +61,14 @@ The easier and safer way to extend functionality is to [override some or all of
 
 1. If you want to replace functions in the ECP module, you need to implement the mandatory utility functions:
 
-These are functions that do not have a counterpart in the standard mbed TLS implementation and their only purpose is to facilitate the integration of the accelerated functions:
-    - `mbedtls_internal_ecp_grp_capable`: implement it to tell mbed TLS if the cryptographic hardware can handle the group. If the answer is no, then mbed TLS will fall back to the software implementation to continue the operation.
-    - `mbedtls_internal_ecp_init` and `mbedtls_internal_ecp_free` are optional. Use them to optimise if you are replacing a function in the ECP module.
-    - For more information about the utility functions read the subsection about the [ECP](#How-to-implement-ECP-module-functions) module.
+    These are functions that do not have a counterpart in the standard mbed TLS implementation and their only purpose is to facilitate the integration of the accelerated functions:
+        - `mbedtls_internal_ecp_grp_capable`: implement it to tell mbed TLS if the cryptographic hardware can handle the group. If the answer is no, then mbed TLS will fall back to the software implementation to continue the operation.
```

Comment:
Please capitalize "implement" for consistent capitalization after a colon across documentation.